src/ocamlorg_frontend/pages/blog.eml

```diff
@@ -13,14 +13,14 @@ Layout.render
                 <h2 class="text-lg mt-0 font-semibold border-b border-current text-primary-600 pb-3">OCaml Community Blog</h2>
 
                 <div class="mt-12 space-y-12">
-                    <% featured |> List.iter (fun (item : Ood.Rss.t) -> %>
+                    <% featured |> List.iter (fun (item : Ood.Planet.t) -> %>
                     <div>
                       <div class="font-semibold text-gray-400">FEATURED</div>
                       <div class="flex flex-col-reverse lg:flex-row lg:gap-12">
                           <div class="space-y-3 mt-6 lg:mt-0 flex flex-col flex-1">
-                              <a href="<%s item.url %>"
+                              <a href="<%s Url.blog_post @@ Option.value ~default:item.slug item.url %>"
```

Comment:
On first glance, I disagree with this. We should not be rendering the Planet blog posts on ocaml.org. Linking out to the blogs is more respectful to the community, and the posts will likely also be more readable in the context of their original page.

I recommend we don't make this change together with the Changelog addition. Pulling in the community blog posts should be a separate PR - if this is even something that we and the community want. Reason: this is a feature unrelated to the changelog and it will be helpful to have a separate diff for that patch.

Also, the URLs are broken: I see http://127.0.0.1:8080/blog/https://tarides.com/blog/2023-03-02-the-journey-to-ocaml-multicore-bringing-big-ideas-to-life

Reply:
To clarify, this is not rendering community blog posts on OCaml.org, we still link to the websites if there's a link, in particular for all the posts we fetched from the RSS feed. We only render the blog posts that are hosted on OCaml.org, for instance, the post on signing opam-repository.

Reply:
Ah ok, then I was misreading that. Only URL needs to be fixed then

Reply:
And IMO this should be part of this PR: moving the opam and OCaml releases out of `news/` to the Changelog makes the current news section confusing, with essentially blog posts with the Multicore newsletter.

I can make two PRs if it's easier to review but from a user perspective making the news section of the blog clearer should be part of the changelog introduction I think.

Reply:
Fixed

Reply:
Yes, it's working now and good to go.